examples/remote_procedure/mnist/run_task.py

```diff
@@ -65,7 +65,7 @@ def handle_with_model(
     )
 
     task_dir = cfg.task_dir
-    build_custom_bundle(task_dir)
+    build_custom_bundle(task_dir, cfg.mephisto)
```

Comment:
We decided to turn this into an overloaded method with default values, so as to avoid introducing breaking changes. I'm happy to use Python's kwargs here for the remaining two params, `post_build_script` (default `""`) and `force_rebuild` (default: `False`)

**Example usage:**
```
build_custom_bundle(task_dir) # still workds
build_custom_bundle(task_dir, force_rebuild=True)
build_custom_bundle(task_dir, force_rebuild=cfg.mephisto.task.force_rebuild) # from config
```

@JackUrb @Etesam913